modbus_south.cpp

```diff
@@ -1567,34 +1570,42 @@ ModbusCacheManager	*manager = ModbusCacheManager::getModbusCacheManager();
 	else if (manager->isCached(m_slave, MODBUS_REGISTER, m_map->m_registerNo))
 	{
 		regValue = manager->cachedValue(m_slave, MODBUS_REGISTER, m_map->m_registerNo);
+		double decValue;
+
+		Logger::getLogger()->error("Check 2:regValue is %d", regValue);
```

Comment:
Replace with Logger::getLogger()->debug

Reply:
Same with this, this Logger line should have been removed.